{{ cookiecutter.project_name }}/Dockerfiles/Dockerfile_python3

```diff
@@ -0,0 +1,8 @@
+FROM python:3
```

Comment:
I am not sure where `python:3` is a good image to use. Does this really maintain the exact python3-version? Or might it switch between 3.8 and 3.7 etc.? I think you should add an option here in the setup of the repo which exact python-version that is used. In some way, determine which python-version is used when creating the repository and put that as the version here.